packages/design-system/src/components/Tabs/Tabs.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useRef } from 'react';
 import Tab from './Tab';
-import TabPanel from './TabPanel';
+import TabPanel, { TabPanelProps } from './TabPanel';
```

Comment:
Curious why `TabPanel` wasn't also made into a named import here? Because it's also the default export?

Reply:
Ah, just because it didn't affect what I was doing. This module changed because of an error that came up in the TS upgrade, but all the icon changes were so that the types in those modules would make it into the design system exports so they could be accessed from `ds-medicare-gov`

Reply:
I didn't make a conscious decision about the exports here